index.html

```diff
@@ -4,11 +4,11 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
+  
   <title>Hello Microverse</title>
 </head>
 <body>
-  <h1 class="header">Hello World!</h1>
+  <h1 class="header">Hello</h1>
```

Comment:
kindly remove one line from your index.html because it is recommended to only have one h1 tag 

  <h1 class="header">Hello World!</h1>
  <h1 class="header">Hello</h1>